pages/client_page.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+    <div class="container">
+        <button>&lt;&ensp; Powrót</button>
```

Comment:
nie można przechodzić między podstronami - fajnie byłoby dodać routing żeby chociaż móc się wracać - można to zrobić na przykład za pomocą `<nuxt-link to="/"></nuxt-link>`

Reply:
nie wiadomo do czego to powrót